pytest_plugins/infra_dependent_markers.py

```diff
@@ -0,0 +1,15 @@
+# Custom infra dependent markers for selection and deselection of tests
+
+
+def pytest_configure(config):
+    """Register custom markers to avoid warnings."""
+    markers = [
+        "on_premises_provisioning: Tests that runs on on_premises Providers",
+        "libvirt_discovery: Tests depends on Libvirt Provider for discovery",
+        "libvirt_content_host: Tests depends on Libvirt Provider for content hosts",
+        "external_auth: External Authentication tests",
+        "vlan_networking: Tests depends on static predefined vlan networking etc",
+    ]
+    markers.extend(markers)
```

Comment:
Why is this line here, seems like it would double the list content?

Reply:
Yep, this does in fact double the registered markers: 

```
@pytest.mark.on_premises_provisioning: Tests that runs on on_premises Providers

@pytest.mark.libvirt_discovery: Tests depends on Libvirt Provider for discovery

@pytest.mark.libvirt_content_host: Tests depends on Libvirt Provider for content hosts

@pytest.mark.external_auth: External Authentication tests

@pytest.mark.vlan_networking: Tests depends on static predefined vlan networking etc

@pytest.mark.on_premises_provisioning: Tests that runs on on_premises Providers

@pytest.mark.libvirt_discovery: Tests depends on Libvirt Provider for discovery

@pytest.mark.libvirt_content_host: Tests depends on Libvirt Provider for content hosts

@pytest.mark.external_auth: External Authentication tests

@pytest.mark.vlan_networking: Tests depends on static predefined vlan networking etc

```

Reply:
Was in hurry .. and made mistakes ! :)